src/serialize/json/normalize.ts

```diff
@@ -459,7 +460,9 @@ export class LicenseNormalizer extends BaseJsonNormalizer<Models.License> {
         text: data.text === undefined
           ? undefined
           : this._factory.makeForAttachment().normalize(data.text, options),
-        url: data.url?.toString()
+        url: data.url === undefined
```

Comment:
I forgot to add the `JsonSchema.isIriReference()` here.
could you add it?
if it returned false, the the url should be omitted

Reply:
done via https://github.com/CycloneDX/cyclonedx-javascript-library/pull/996/commits/95fb7c92e28f9bf5ed8f2378ad550f41d2f4d0bf